packages/deploy-aws/src/seagull_stack.ts

```diff
@@ -18,6 +18,7 @@ import { getApiGatewayDomain, getApiGatewayPath } from './lib'
  */
 
 export class SeagullStack extends Stack {
+  defaultRole?: IAM.Role // for extensions
```

Comment:
I know I suggested this change, but I would recommend to put it within the app, instead of the stack, because the stack straigth forward about adding resources. I would not recommend to blow it up with complexer structures.

Reply:
It's moved to app now